lib/network.py

```diff
@@ -1058,26 +1076,27 @@ def init_headers_file(self):
             b.update_size()
 
     def run(self):
-        self.init_headers_file()
+        self._init_headers_file()
         while self.is_running():
-            self.maintain_sockets()
-            self.wait_on_sockets()
-            self.maintain_requests()
+            self._maintain_sockets()
+            self._wait_on_sockets()
+            self._maintain_requests()
             self.run_jobs()    # Synchronizer and Verifier
-            self.process_pending_sends()
-        self.stop_network()
+            self._process_pending_sends()
+        self._stop_network()
         self.on_stop()
 
-    def on_notify_header(self, interface, header_dict):
+    def _on_notify_header(self, interface, header_dict):
         try:
             header_hex, height = header_dict['hex'], header_dict['height']
         except KeyError:
             # no point in keeping this connection without headers sub
             self.connection_down(interface.server)
```

Comment:
_connection_down

Reply:
there's also one at (new) line 941, but it's too far from changed lines so I can't comment there